scripts/test

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-NODE_ENV=test node_modules/.bin/karma start "$@"
+# NODE_ENV=test node_modules/.bin/karma start "$@"
```

Comment:
[Minor] We're on version-control, so why not just remove the line 😄

Reply:
Having it commented out is a more passive aggressive way of saying "you should sort out your tests" 😉